client/src/common/components/MessageBox.jsx

```diff
@@ -6,7 +6,7 @@ import { MessageType } from 'common/constants/enums';
 import { MessagePropType } from 'common/constants/propTypes';
 import { InfoIcon } from 'assets/images/icons';
 
-const MessageBox = ({ message, type }) => (
+const MessageBox = ({ message, type, children }) => (
```

Comment:
I think you should drop `message` prop and always use the `children`. This requires changes across all usages. Also, please consider sorting the props.